Description: link Python gRPC module with all needed Abseil libraries
 Add the needed libraries to link with.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: https://bugs.debian.org/1026937
Forwarded: no
Last-Update: 2026-07-11

---

--- a/setup.py
+++ b/setup.py
@@ -292,7 +292,7 @@ if EXTRA_ENV_LINK_ARGS is None:
             EXTRA_ENV_LINK_ARGS += " -latomic"
     if "linux" in sys.platform:
         EXTRA_ENV_LINK_ARGS += " -static-libgcc"
-        EXTRA_ENV_LINK_ARGS += ' -labsl_str_format_internal -labsl_time'
+        EXTRA_ENV_LINK_ARGS += ' -labsl_str_format_internal -labsl_time -labsl_hash -labsl_raw_hash_set -labsl_status -labsl_statusor -labsl_random_internal_randen_slow -labsl_random_internal_randen_hwaes_impl -labsl_random_internal_randen -labsl_random_internal_entropy_pool -labsl_log_initialize'
 
 # Explicitly link Core Foundation framework for MacOS to ensure no symbol is
 # missing when compiled using package managers like Conda.
